website/source/guides/web-ui/operating-a-job.html.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: "guides"
+page_title: "Operating a Job from the Web UI"
+sidebar_current: "guides-web-ui-operating-a-job"
+description: |-
+  Learn how to operate a job from the Web UI.
+---
+
+# Reviewing Job and Allocation Status
+
+The Web UI can be a powerful companion when monitoring and debugging jobs running in Nomad. The Web
+UI will list all jobs, link jobs to allocations, allocations to client nodes, client nodes to driver
+health, and much more. This creates a fluid exploratory experience.
```

Comment:
Perhaps take out the last sentence **This creates a fluid exploratory experience.**